geometry_msgs/PoseStamped target_pose
bool cartesian_path
---
bool success
string message
